package.json

```diff
@@ -65,7 +72,6 @@
     "node-sass": "^4.5.2",
     "octicons": "^6.0.1",
     "parallel-webpack": "^2.1.0",
-    "prettier": "1.7.4",
```

Comment:
Actually, I'd rather we keep as much stuff in `dependencies` as possible. The `electron` packages have been moved down because `electron-builder` demands it, and the `@types/` packages are just declaration files.